hazelcast/src/test/java/com/hazelcast/client/config/ClientConfigXmlGeneratorTest.java

```diff
@@ -777,6 +777,14 @@ public void testSqlConfig() {
         assertEquals(originalConfig.getResubmissionMode(), generatedConfig.getResubmissionMode());
     }
 
+    @Test
+    public void testAltoConfig() {
+        ClientAltoConfig originalConfig = new ClientAltoConfig().setEnabled(true);
+        clientConfig.setAltoConfig(originalConfig);
+        ClientAltoConfig generatedConfig = newConfigViaGenerator().getAltoConfig();
+        assertEquals(originalConfig.isEnabled(), generatedConfig.isEnabled());
```

Comment:
Why not use `assertTrue(generatedConfig.equals(originalConfig))` ? for future changes, we would not need to update this test